Libraries/LibGUI/Clipboard.cpp

```diff
@@ -79,10 +100,18 @@ void Clipboard::set_data(const StringView& data, const String& type)
     WindowServerConnection::the().send_sync<Messages::WindowServer::SetClipboardContents>(shared_buffer->shbuf_id(), data.length(), type);
 }
 
+void Clipboard::set_data(const RefPtr<Gfx::Bitmap> bitmap, const String& type)
+{
+    auto bitmap_with_shared_buffer = bitmap->to_bitmap_backed_by_shared_buffer();
+    auto shared_buffer = bitmap_with_shared_buffer->shared_buffer();
+    shared_buffer->share_with(WindowServerConnection::the().server_pid());
+
+    WindowServerConnection::the().send_sync<Messages::WindowServer::SetClipboardContents>(shared_buffer->shbuf_id(), shared_buffer->size(), type);
+}
+
 void Clipboard::did_receive_clipboard_contents_changed(Badge<WindowServerConnection>, const String& data_type)
 {
     if (on_content_change)
         on_content_change(data_type);
 }
-
```

Comment:
This seems like an unnecessary whitespace change that `clang-format` should not do. Are you using the project's `.clang-format` configuration file (found in the project root)?

Reply:
I'm using the one on the root (or that is what I configured) but that somewhat slipped in there. Formatting it again removed that blank line so I don't know when I introduced that there. Anyway now it isn't there